### ISSUE TITLE
Hide slug field from the user

### ISSUE THREAD

**Issue by jesperlindstrom** [cthit/chalmersit-rails#219]
On pages and committees there's a "slug" field. Far from everyone understands what a slug is and at the same time, this can easily be auto-generated from the title. I suggest that we:

1. Put the slug field below the title and use JS to fill in the slug field, from the title, as one is typing in the title field.

2. Clarify what a slug is. Perhaps simply put the slug text field *inside* of the URL, like this:
> **Committee URL:**
> chalmers.it/committees/[slug text field here]

alternatively, we can just remove the slug field and auto-generate it from the title / committee name. Committee names are unique by nature and duplicates should not be allowed to begin with.

**Comment by lindskogen** [comment]
Slug is not intended to be visible for !admins. Maybe "dumb it down" and only show title + body?

**Comment by jesperlindstrom** [comment]
@lindskogen yeah that's even better. Only show what the user cares about, and handle the rest (such as the slug) automatically.
